src/main/java/de/tum/cit/aet/artemis/atlas/web/admin/AdminStandardizedCompetencyResource.java

```diff
@@ -38,6 +38,8 @@
  * Admin REST controller for managing {@link StandardizedCompetency} entities.
  */
 @Profile(PROFILE_CORE)
+@FeatureToggle(Feature.StandardizedCompetencies)
```

Comment:
Seems like this change is causing exceptions when the tests access the corresponding endpoints. Can you have a look?

Reply:
There was a bug in the FeatureToggleAspect that I fixed now